inference/inference.py

```diff
@@ -199,14 +195,18 @@ def autoregressive_inference(params, ic, valid_data_full, model):
     if params.masked_acc:
       maskarray = torch.as_tensor(np.load(params.maskpath)[0:720]).to(device, dtype=torch.float)
 
-    valid_data = valid_data_full[ic:(ic+prediction_length*dt+n_history*dt):dt, in_channels, 0:720] #extract valid data from first year
+    valid_data = valid_data_full[ic:(ic+prediction_length*dt+n_history*dt):dt, in_channels] #extract valid data from first year
+    if valid_data.shape[2] > 720:
+       # might be necessary for ERA5 data
+       valid_data = valid_data[:, :, 0:720]
+       
     # standardize
     valid_data = (valid_data - means)/stds
     valid_data = torch.as_tensor(valid_data).to(device, dtype=torch.float)
 
     #load time means
     if not params.use_daily_climatology:
-      m = torch.as_tensor((np.load(params.time_means_path)[0][out_channels] - means)/stds)[:, 0:img_shape_x] # climatology
+      m = torch.as_tensor((time_means[out_channels] - means)/stds)[:, 0:img_shape_x]
```

Comment:
Does this no longer have to do with "climatology?"

Reply:
Awesome that this is no longer so hard coded!

Reply:
Well, the "time means" statistic is a naive climatology (one without any seasonal cycle).